docker-compose.yaml

```diff
@@ -58,8 +58,8 @@ services:
 
   recomendator:
     build:
-      context: .
-      dockerfile: recomendator/Dockerfile
+      context: recomendator/
```

Comment:
why is this appearing as a change if it's already in the splitting-the-monlith branch?

Reply:
The change was not applied in this branch cause git things XD